package.json

```diff
@@ -20,8 +21,11 @@
     "start:cloud-viewer": "cross-env SINGLE_START=camunda-cloud-viewer npm run dev",
     "start:platform-navigated-viewer": "cross-env SINGLE_START=camunda-platform-navigated-viewer npm run dev",
     "start:cloud-navigated-viewer": "cross-env SINGLE_START=camunda-cloud-navigated-viewer npm run dev",
-    "prepublishOnly": "run-s test:distro",
-    "prepare": "run-s clean distro"
+    "prepublishOnly": "run-s generate-types test:distro",
+    "prepare": "run-s clean distro",
+    "generate-types": "run-s generate-types:*",
+    "generate-types:generate": "del-cli \"lib/**/*.d.ts\" && npx bio-dts -r --resolveJsonModule --esModuleInterop lib",
```

Comment:
Do we want to depend on an explicit version of `bio-dts`, i.e. through installation:

```sh
npm install bio-dts -D
```

Reply:
If we decide to, we should make it explicit in diagram-js and bpmn-js, too.